modules/nf-core/last/lastal/meta.yml

```diff
@@ -1,41 +1,43 @@
-name: last_lastal
-description: Find suitable score parameters for sequence alignment
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "last_lastal"
+description: Aligns query sequences to target sequences indexed with lastdb
 keywords:
   - LAST
   - align
   - fastq
   - fasta
 tools:
-  - last:
-      description: LAST finds & aligns related regions of sequences.
-      homepage: https://gitlab.com/mcfrith/last
-      documentation: https://gitlab.com/mcfrith/last/-/blob/main/doc/last-train.rst
-      tool_dev_url: https://gitlab.com/mcfrith/last
+  - "last":
```

Comment:
Again, why do these have quote marks?

Reply:
Same reason here.  Please note that some other modules also have the quotes.